packages/react-components/react-select/src/stories/Select/index.stories.tsx

```diff
@@ -1,4 +1,4 @@
-import { Select } from '../Select';
+import { Select } from '@fluentui/react-select';
 import selectDescription from '../SelectDescription.md';
```

Comment:
Think you need to move `SelectDescription.md` file to the `stories/Select` folder as well